docs/book/src/plugins/available-plugins.md

```diff
@@ -1,5 +1,15 @@
-# Available Plugins
+# Available plugins
 
-This section details how to use the currently available plugins in Kubebuilder. 
+This section describes the plugins supported and shipped in with the Kubebuilder project.
 
-  - [Declarative V1](declarative-v1.md)
\ No newline at end of file
+<aside class="note">
+
+<h1>But you can create your own plugins, see:</h1>
```

Comment:
Hun, the page speaks about available plugins. then, could we not consider this content a note, or observation? Because of this in POV it shows fit better in the <aside>